src/main/java/org/apache/maven/archiver/PomPropertiesUtil.java

```diff
@@ -87,8 +87,9 @@ private void createPropertiesFile(Properties properties, Path outputFile, boolea
             }
 
             Collections.sort(lines);
-            for (String l : lines) {
-                pw.println(l);
+            for (String line : lines) {
+                writer.write(line);
+                writer.write( '\n' );
```

Comment:
`System.lineSeparator()` ?

Reply:
Moot. This code can be deleted now that we're using properties.store